### PR TITLE
Add some monitoring components from deployer

### PR DESCRIPTION
**Pull request by mroman-redhat**

### CODE REVIEW

File: jupyterhub/jupyterhub/base/jupyterhub-db-probe.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+          name: metrics
+        resources:
+          limits:
+            cpu: 300m
+            memory: 50Mi
+          requests:
+            cpu: 300m
+            memory: 50Mi
+
+---
```
Comment (anishasthana) [inline jupyterhub/jupyterhub/base/jupyterhub-db-probe.yaml:42]: This should be split into two separate objects right?

**Comment by crobby** [comment]
Have you tried this out yet?  No issues with some objects being in a different (different than redhat-ods-applications) namespace?  I feel like at one time there may have been some weirdness if we tried to use multiple namespaces in the same KfDef.

**Review by anishasthana — CHANGES_REQUESTED** [review]

File: kfdef/kfctl_openshift.yaml [hunk 2]
```diff
@@ -110,6 +110,11 @@ spec:
         name: manifests
         path: odh-dashboard
     name: odh-dashboard
+  - kustomizeConfig:
```
Comment (anishasthana) [inline kfdef/kfctl_openshift.yaml:113]: I'd remove this from the sample kfctl file

**Review by anishasthana — COMMENTED** [review]
It looks like to deploy everything in here you'd basically need two new kfdefs, unfortunately. One for the cluster monitoring rbac, and one for the monitoring objects. The db probe itself will be deployed with the usual kfdef we use.

File: kfdef/kfctl_openshift.yaml (hunk 2 above)
Comment (crobby) [reply kfdef/kfctl_openshift.yaml:113]: Yeah, it's not really used anywhere in red-hat-data-services and might be an unnecessary diff from opendatahub-io

**Comment by crobby** [comment]
I think this is good.  I do agree with Anish that maybe we don't need the addition to the sample kfdef.  Other than that, we should be able to get these merged.

**Review by crobby — APPROVED** [review]

**Comment by crobby** [comment]
@anishasthana Are you good with this now?

**Review by anishasthana — APPROVED** [review]
/lgtm

We need to make sure to merge the corresponding deployer PR in as well.

**Pull request merged**